other/get.webgl.org/webgl2/index.html

```diff
@@ -182,18 +182,44 @@
     addClass($$("no-javascript"), "webgl-hidden");
     b = BrowserDetect;
     b.init();
+    var badImpl = false;
     var canvas = document.getElementById("webgl-logo");
     try { gl = canvas.getContext("webgl2"); }
     catch (x) { gl = null; }
 
+    if (gl) {
+        // check if it really supports WebGL2. Issues, Some browers claim to support WebGL2
+        // but in reality pass less than 20% of the conformance tests. Add a few simple
+        // tests to fail so as not to mislead users.
+        var params = [
+            { pname: 'MAX_3D_TEXTURE_SIZE', min: 256, },
+            { pname: 'MAX_DRAW_BUFFERS', min:4, },
+            { pname: 'MAX_COLOR_ATTACHMENTS', min:4, },
+            { pname: 'MAX_VERTEX_UNIFORM_BLOCKS', min:12, },
+            { pname: 'MAX_VERTEX_TEXTURE_IMAGE_UNITS', min:16, },
+            { pname: 'MAX_FRAGMENT_INPUT_COMPONENTS', min:60, },
+            { pname: 'MAX_UNIFORM_BUFFER_BINDINGS', min:24, },
+            { pname: 'MAX_COMBINED_UNIFORM_BLOCKS', min:24, },
+        ];
+        for (var i = 0; i < params.length; ++i) {
+            var param = params[i];
+            var value = gl.getParameter(gl[param.pname]);
+            if (typeof value !== 'number' || Number.isNaN(value) || value < params.min) {
```

Comment:
`Number.isNaN(value)` is technically redundant to `value < params.min`, isn't it?

Reply:
<img width="127" alt="Screen Shot 2019-04-24 at 16 09 27" src="https://user-images.githubusercontent.com/234804/56666004-6915fc00-66ab-11e9-9ad1-e7935308b447.png">

Reply:
Ah, yeah, ok.